doc/reference_input.xml

```diff
@@ -0,0 +1,1822 @@
+<?xml version="1.0" encoding="UTF-8"?>
+  <sect1 id="Geometry_Inputs">
+
+    <sect1info>
+			<abstract>
+			<para>These functions create geometry objects from various textual or binary formats.</para>
+			</abstract>
+			</sect1info>
+
+	  <title>Geometry Input</title>
+
+    <sect2>
+      <title>Well-Known Text (WKT)</title>
+
+		<refentry id="ST_BdPolyFromText">
+		  <refnamediv>
+			<refname>ST_BdPolyFromText</refname>
+
+			<refpurpose>Construct a Polygon given an arbitrary collection of closed
+			linestrings as a MultiLineString Well-Known text representation.</refpurpose>
+		  </refnamediv>
+
+		  <refsynopsisdiv>
+			<funcsynopsis>
+			  <funcprototype>
+				<funcdef>geometry <function>ST_BdPolyFromText</function></funcdef>
+				<paramdef><type>text </type> <parameter>WKT</parameter></paramdef>
+				<paramdef><type>integer </type> <parameter>srid</parameter></paramdef>
+			  </funcprototype>
+			</funcsynopsis>
+		  </refsynopsisdiv>
+
+		  <refsection>
+			<title>Description</title>
+
+			<para>Construct a Polygon given an arbitrary collection of closed
+			linestrings as a MultiLineString Well-Known text representation.</para>
+
+			<!-- optionally mention that this function uses indexes if appropriate -->
+			<note>
+				<para>Throws an error if WKT is not a MULTILINESTRING. Throws an
+			error if output is a MULTIPOLYGON; use ST_BdMPolyFromText in that case, or
+			see ST_BuildArea() for a
+			postgis-specific approach.</para>
+			</note>
+
+			 <para>&sfs_compliant; s3.2.6.2</para>
+			 <para>Performed by the GEOS module.</para>
+			 <para>Availability: 1.1.0</para>
+		  </refsection>
+
+		  <refsection>
+			<title>Examples</title>
+
+			<programlisting>Forthcoming</programlisting>
```

Comment:
can we comment out whole section then?

Reply:
Yes, makes sense.

The descriptions for these two functions need a lot of work...